FlymFork/src/main/res/values/strings.xml

```diff
@@ -120,12 +120,12 @@
     <string name="menu_mark_as_unfavorite">Unmark as favorite</string>
     <string name="menu_mark_as_unread">Mark as unread</string>
     <string name="menu_reload_full_text">Reload full text</string>
-    <string name="menu_reload_full_text_without_mobilizer">Reload without mobilizer</string>
-    <string name="share_favorites_title">Flym: starred entries</string>
+    <string name="menu_reload_full_text_without_mobilizer">Reload without Mobilizer</string>
+    <string name="share_favorites_title">Flym Fork: starred entries</string>
     <string name="copied_clipboard">Link copied</string>
-    <string name="get_full_text">Load article full text</string>
-    <string name="show_text_in_entry_list">Show article text in list</string>
-    <string name="original_text">Show original text</string>
+    <string name="get_full_text">Load full article.</string>
+    <string name="show_text_in_entry_list">Show excerpts on article list.</string>
```

Comment:
This is not excerpts, it is article rss full text

Reply:
By the way, yesterday one of my Russian users asked me to add a NEW feature to show serveral first lines of article text in article list - for this word excerpts sounds much better.

Reply:
Livejournal rss channel contains FULL text of article. And so some others. So It is not as easy to decide. Lets leave it ORIGINAL :) for now

Reply:
Ok, let's leave "Show original text on article list.".